userguide/plugins.rst

```diff
@@ -130,17 +134,15 @@ page as shown in :numref:`Figure %s <view_installed_plugins_fig>`.
    Viewing Installed Plugins
 
 
-In addition to the :guilabel:`Jail` name, the :guilabel:`Columns` menu controls
-which information is displayed for each installed Plugins entry. Options include
-status, IPv4 address, IPv6 address, installed version, plugin name, release, boot status,
-and collection.
+In addition to the :guilabel:`Jail` name, the :guilabel:`Columns` menu
+controls which information is displayed for each installed Plugins
+entry.
 
-More information such as *RELEASE*, *AUTOSTART*, and
-*VERSION* is shown by clicking |ui-chevron-right|. Options to
-:guilabel:`RESTART`, :guilabel:`STOP`, :guilabel:`UPDATE`,
-:guilabel:`MANAGE`, and :guilabel:`UNINSTALL` the plugin are also
-displayed. If an installed plugin has notes, the notes can be viewed by
-clicking :guilabel:`POST INSTALL NOTES`.
+More information such as *RELEASE* and *VERSION* is shown by clicking
+|ui-chevron-right|. Options to :guilabel:`RESTART`, :guilabel:`STOP`,
```

Comment:
Avoid making lists of specific action buttons or menu entries. These are on the screen, and having static lists in the doc just means it will be out of sync when the program changes.  In this case, it would be enough to say that there are buttons to control and manage the plugin.

Reply:
This text has existed in angulargui for roughly 4-5 months (https://github.com/freenas/freenas-docs/blame/angulargui/userguide/plugins.rst), so this alteration is outside the scope of this ticket and PR. Please make a Jira ticket or Epic with the specific rework request so this concept can be prioritized then applied uniformly to the entire guide, rather than playing "whack a mole" ad infinitum.